### ISSUE TITLE
Map of "Sikhw desert E" doesn't fit neighbours

### ISSUE THREAD

**Issue by ghost** [arianne/stendhal#387]
## 🐞 Describe the bug

While in the SE corner of the "Sikhw desert E" map, when going to the eastern neighbour zone, one is unable to come back because there's a fence between the zones.
Also, when going to the southern neighbour zone, the player positions gets shifted to the west because there are some trees between the zones.

## 🔥 How to reproduce

Steps to reproduce the first behavior:
1. Go to 0_sikhw_desert_e at (127,127)
2. Go to the right
3. Find yourself in Fado, unable to go back to the left

Steps to reproduce the first behavior:
1. Go to 0_sikhw_desert_e at (127,127) - this is exactly in the corner
2. Go down
3. Find yourself in "Kalavan castle W", but a bit to the left, not exactly in the corner, as expected

## 🌍  Environment

- Java client

Location in game: 0_sikhw_desert_e at (127,127)

## 🛠️ Suggested fixes

Ideas on how to fix the problem:

1. Between "Sikhw desert E" and "Fado city"
    - add some trees to the entire eastern border of "Sikhw desert E"
    - or move the western fence of the "Fado city" farm a bit to the east
    - or remove the western fence of the "Fado city" farm altogether
2. Between "Sikhw desert E" and "Kalavan castle W"
    - add some trees in the SE corner of "Sikhw desert E", to match the ones from the NE corner of "Kalavan castle W"

## 🔭 Additional information

Screenshot of the "Sikhw desert E" minimap with the relevant neighbours:
![4minimaps](https://user-images.githubusercontent.com/42698386/90428283-6c1cd300-e0cc-11ea-80d1-dd14c68497d3.png)

Screenshot of before and after going from "Sikhw desert E" to "Fado city":
![west_east](https://user-images.githubusercontent.com/42698386/90429969-4e9d3880-e0cf-11ea-9ef9-f826b4eb50c5.png)

**Comment by AntumDeluge** [comment]
Thank you for the report @ghost.

**Comment by nhnb** [comment]
I am sorry, I did not notice this report.

The mountain was very carefully design around the map boarders to look well on the world map while at the same time preserving the name "Kavalan" when they approaching it from the east. 

Therefore we should block the northern path in "Kalavan castle W", so that players are unable to reach the map "Sikhm desert E".

I am quite surprised, that there are gaps in the collision layer of the mountain levels in "Kalavan castle W". Are we still sure, that players cannot enter the Sikhm region?

**Comment by AntumDeluge** [comment]
@nhnb I'm not very familiar with the region of Sikhw. In 0_sikhw_desert_e, I added a fence with collision to make it look like Fado city's border. I also added some collision to the bottom-right corner of the desert with some trees to match the collision of 0_kalavan_castle_west to the south. That fixes the issues reported here.

![0_sikhw_desert_e_new_collision](https://user-images.githubusercontent.com/3631473/146838727-06d4fd52-0cd5-4473-ad20-a2a8bda3dc21.png)

If you walk to the south from 0_sikhw_desert_e, you hit 0_kalavan_castle_west. There is an opening in 0_kalavan_castle_west that leads to 0_kalavan_castle. Should that not be the case? From what you said, it sounds like players should not be able to enter Sikhw from Kalavan at all.
